tensorboard/webapp/webapp_data_source/tb_feature_flag_data_source.ts

```diff
@@ -98,6 +98,23 @@ export class QueryParamsFeatureFlagDataSource
     return featureFlags;
   }
 
+  storeFeatureFlag(flagKey: string, value: boolean) {
+    if (value) {
+      localStorage.setItem(flagKey, 'true');
+    } else {
+      localStorage.setItem(flagKey, 'false');
+    }
+  }
+
+  getPersistentFeatureFlagState(flagKey: string): boolean | null {
```

Comment:
Could use some consistency in your API name. `storeFeatureFlag` is not at all obviously the inverse of `getPersistentFeatureFlagState`. Also I don't like the use of "State" here since State has a specific meaning in ngrx world.  

Maybe:

`persistFeatureFlag`/`getPersistentFeatureFlags`